fish_passage_fieldwork.Rmd

```diff
@@ -0,0 +1,37 @@
+# Fish Passage Field Work Guide
+
+## Field Work Prep
```

Comment:
Safety should be first here. There are lots of things that happen (not exaustive list - field safety plan, charge hand radios, charge inreach/zoleo, pilons, high vis vests, ski poles for stream walks, bear spray, check fire information, portable speaker to warn bears, whistle).  Equipment list should contain checks for this work for all the stuff we use. Thinking of talk to texting openAI to help build an extra column that says the "why" for each thing on there but there is only so many hours in the day.....